src/components/ChatList/ChatItemFunc.tsx

```diff
@@ -60,22 +67,59 @@ export default function ChatItemFunc({id, name}: {id: string, name: string}) {
 
   return (
     <>
-      <IconButton ref={referenceElem} icon='dots' onClick={openPopper} classNames='edit-chat-btn'/>
-      {isShown && !isEditMode && (
-        <PopperDialog onBlur={close} referenceElem={referenceElem.current}>
-          <StyledLink ref={referenceRenameElem} preIcon='edit' onClick={renameChat}>Rename</StyledLink>
-          <StyledLink preIcon='remove' onClick={removeChat}>Delete</StyledLink>
-        </PopperDialog>
-      )}
-      {isEditMode && (
-        <PopperDialog onBlur={closeChatRename} referenceElem={referenceRenameElem.current} style={{padding: '12px'}}>
-          <StyledInput type='text' name='chatname' value={chatName} onChange={handleChange} />
-          <div className='edit-func'>
-            <StyledButton onClick={discardEdit}>Discard</StyledButton>
-            <StyledButton onClick={saveEdit} classNames='primary'>Save</StyledButton>
-          </div>
-        </PopperDialog>
-      )}
+
+      {isChatPage ?
```

Comment:
- [ ] remove isChatPage. if you have 2 different behaviors - just move each of the to a separate component